magma/backend/coreir_transformer.py

```diff
@@ -192,7 +192,9 @@ def connect_non_outputs(self, module_defn, port, non_input_ports):
             for elem in port:
                 self.connect_non_outputs(module_defn, elem, non_input_ports)
         elif not port.is_output():
-            self.connect(module_defn, port, port.value(), non_input_ports)
+            if port.trace() is None:
```

Comment:
stray `print`? if we want to have this information we should use `logger.info()`